soql-analyzer/src/test/scala/com/socrata/soql/analyzer2/SoQLAnalyzerTest.scala

```diff
@@ -758,6 +758,33 @@ class SoQLAnalyzerTest extends FunSuite with MustMatchers with TestHelper {
     analysis1.statement must be (isomorphicTo(analysis2.statement))
   }
 
+  test("Hidden columns - order by hidden") {
+    val tf1 = tableFinder(
+      (0, "aaaa-aaaa") -> D("n1" -> TestNumber, "n2" -> TestNumber),
+      (0, "bbbb-bbbb") -> Q(0, "aaaa-aaaa", "select n1, n2*2 as x order by x").withHiddenColumns("x")
+    )
+    val analysis1 = analyze(tf1, "bbbb-bbbb", "select :*, *")
+
+    val tf2 = tableFinder(
+      (0, "aaaa-aaaa") -> D("n1" -> TestNumber, "n2" -> TestNumber),
+      (0, "bbbb-bbbb") -> Q(0, "aaaa-aaaa", "select n1 order by n2*2")
+    )
+    val analysis2 = analyze(tf2, "bbbb-bbbb", "select :*, *")
+
+    analysis1.statement must be (isomorphicTo(analysis2.statement))
+  }
+
+  test("Hidden columns - union") {
+    val tf1 = tableFinder(
+      (0, "aaaa-aaaa") -> D("n1" -> TestNumber, "n2" -> TestNumber).withHiddenColumns("n1"),
+      (0, "bbbb-bbbb") -> D("n3" -> TestNumber),
+      (0, "cccc-cccc") -> Q(0, "aaaa-aaaa", "select * union select n3 from @bbbb-bbbb"),
+    )
+    val analysis1 = analyze(tf1, "cccc-cccc", "select :*, *")
+
+    println(analysis1.statement.debugDoc)
```

Comment:
Debug?

Reply:
Yep, oops.  This entire class's change was not meant to be checked in.